tests/TestHelpers/GraphHelper.php

```diff
@@ -537,6 +539,41 @@ public static function addUsersToGroup(
 		);
 	}
 
+	/**
+	 * tries to add group to a group
+	 *
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $adminUser
+	 * @param string $adminPassword
+	 * @param string $groupId
+	 * @param array $userId
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function addGroupToGroup(
+		string $baseUrl,
+		string $xRequestId,
+		string $adminUser,
+		string $adminPassword,
+		string $groupId,
+		string $userId
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'groups/' . $groupId);
+		$payload = [ "members@odata.bind" => [] ];
+		$payload["members@odata.bind"][] = self::getFullUrl($baseUrl, 'groups/' . $userId);
```

Comment:
we have used this for single user request
```suggestion
		$payload = [
			"@odata.id" => self::getFullUrl($baseUrl, 'groups/' . $userId)
		];
```